install_config/upgrading/manual_upgrades.adoc

```diff
@@ -748,6 +748,37 @@ ifdef::openshift-origin[]
 ====
 endif::[]
 
+After a manual upgrade, get the latest templates from *openshift-ansible-roles*:
+
+====
+----
+rpm -ql openshift-ansible-roles | grep examples | grep v1.2
+----
+====
+
+In this example,
+*_/usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.2/image-streams/image-streams-rhel7.json_*
+is the latest file that you want in the latest *openshift-ansible-roles* package.
+
+*_/usr/share/openshift/examples/image-streams/image-streams-rhel7.json_* is not
+owned by a package. It is updated by Ansible. If you are upgrading outside of
+Ansible. you need to get the latest .json files on the system where you are
+running `oc`, which can run anywhere that has access to the master.
+
+Install *atomic-openshift-utils* and its dependencies to install the new content
+into
+*_/usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.2/_*.:
+
+====
+----
+$ oc create -n openshift -f  /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.2/image-streams/image-streams-rhel7.json
```

Comment:
Unfortunately we don't have a good method to create anything that doesn't exist and update things that do exist so we have to create, then replace. Like this 

```
$ oc create -n openshift -f  /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.2/image-streams/image-streams-rhel7.json
$ oc replace -n openshift -f  /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.2/image-streams/image-streams-rhel7.json
```